src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private static final double PI = 3.14;
```

Comment:
The value of PI should be more precise. Consider using 'Math.PI' which is a built-in constant that represents the value of PI more accurately.